ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20__periodics.yaml

```diff
@@ -0,0 +1,115 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_4.20_base-rhel9:
+    name: "4.20"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-8-golang-1.23-openshift-4.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.23-openshift-4.20
+  ocp_builder_rhel-9-golang-1.23-openshift-4.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.23-openshift-4.20
+  upi-installer:
+    name: "4.20"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.23
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: ci
+      version: "4.20"
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.20"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-kubevirt-ovn-techpreview
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: aws
+    env:
+      CNV_PRERELEASE_LATEST_CHANNEL: "true"
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: c5n.metal
+      FEATURE_SET: TechPreviewNoUpgrade
+    workflow: openshift-e2e-aws-ovn-virt
+- as: e2e-aws-ovn-virt-periodic
+  cron: 0 3 * * *
+  steps:
+    cluster_profile: aws-2
+    env:
+      CNV_PRERELEASE_LATEST_CHANNEL: "true"
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: c5n.metal
+    workflow: openshift-e2e-aws-ovn-virt
```

Comment:
OK cool, we need this for our upcoming features - one TP and one GA virt lane.
The GA lane will run the "old" virt feature tests (primary UDN at least) while the GA lane will run those + the new feature tests.